### DISCUSSION TITLE
Symlinks to new folder (RGB-PI OS4)

### DISCUSSION THREAD

**Discussion by noz1380** [retronas/discussions#53] [Q&A]
I currently use RetroNAS for OPL and my PS2, it works super!!

I have an RGB-PI setup at home, and I have updated to OS4.

I want to add my roms to the right folder structure (As defined by the OS)

Can I copy the structure, then create symlinks to the generic roms folder?  I assume that is how its done?  Any good tutorials I can follow?  Happy to use CLI.

**Comment by sairuk** [comment]
you could review [batocera playbook](https://github.com/danmons/retronas/blob/main/ansible/install_batocera_cifs.yml) and templates in the ansible directory to see how we would normally integrate it

most systems consist of a 
1. top level folder in your retronas data directory
1. a samba share file dropped it inplace on the filesystem in /etc/samba pointing to the aforementioned directory
1. symlinks with whatever names the project needs in the new directory linked to the appropriate generic dirs in roms, bios, saves etc.

I didn't see any links to a github project on their site for me to see how RGB-PI is put together, so if you do the work in a playbook we'd happily review it for potential ntegration.

**Reply by noz1380** [reply]
Ok I think I can follow that. 

Just one thing, RBG-Pi uses NFS, so do I still need to do the things you refer to with samba?

**Reply by sairuk** [reply]
ah ok, no you would need to install the retronas nfs service and create an export for your top level directory instead of samba

**Reply by noz1380** [reply]
@sairuk I have set up the folders, and symlinks, finally got my head around it.  Adjusted samba conf so my share shows up in windows.

I'm struggling now to connect the the nfs via the RGB-Pi OS.  

I have mounted it in Ubuntu cli.  So I will ask the question of the RGB-Pi Team.

**Reply by sairuk** [reply]
it would be really nice if you could write up your method for other RetroNAS/RGP-PI users once you have it all sorted, good luck.

**Reply by noz1380** [reply]
Once I understand the Playbook, I'll have a blast at doing one for RGB-Pi.

Happy to try and help out.
